ksql-parser/src/main/java/io/confluent/ksql/schema/ksql/LogicalSchemas.java

```diff
@@ -69,6 +69,12 @@ private LogicalSchemas() {
      * @return the sql type.
      */
     Type toSqlType(Schema schema);
+
+    /**
+     * Ensures that the schema passed in is valid. Otherwise, it throws a
+     * {@code KsqlException}.
+     */
+    void validate(Schema schema) throws KsqlException;
```

Comment:
This feels like API bloat...

Reply:
@rodesai wanted this - I don't really care one way or another.